lib/plugins/redis/lib/Redis.js

```diff
@@ -8,13 +8,14 @@ class Redis {
 
   init(config) {
     this.redis = require('redis')
-
+    
     Promise.promisifyAll(this.redis.RedisClient.prototype)
```

Comment:
we can put timeout options in custom promisifier

Reply:
sure, updated with `customPromisifier`